File: app/src/main/java/cl/monsoon/s1next/view/internal/ToolbarDropDownInterface.java
package cl.monsoon.s1next.view.internal;

import java.util.List;

public interface ToolbarDropDownInterface {

    interface OnItemSelectedListener {

        void onToolbarDropDownItemSelected(int position);
    }

    interface Callback {

        void setupToolbarDropDown(List<? extends CharSequence> dropDownItemList);
    }
}

File: app/src/main/java/me/ykrank/s1next/view/internal/ToolbarDropDownInterface.kt
package me.ykrank.s1next.view.internal

interface ToolbarDropDownInterface {
    interface OnItemSelectedListener {
        fun onToolbarDropDownItemSelected(position: Int)
    }

    interface Callback {
        fun setupToolbarDropDown(dropDownItemList: List<CharSequence>)
    }
}
